CMakeLists.txt

```diff
@@ -3,6 +3,7 @@ cmake_minimum_required(VERSION 3.3)
 # project variables
 project(spdlog_setup VERSION 0.3.0 LANGUAGES CXX)
 set(SPDLOG_MIN_VERSION "1.0.0")
+set(CPPTOML_MIN_VERSION "0.1.1")
```

Comment:
Should set `CPPTOML_MIN_VERSION` to `"0.1.0"`, since 0.1.0 was indeed working previously.